React/Views/RCTWebView.m

```diff
@@ -240,11 +289,11 @@ - (BOOL)webView:(__unused UIWebView *)webView shouldStartLoadWithRequest:(NSURLR
     data = [data stringByReplacingOccurrencesOfString:@"+" withString:@" "];
     data = [data stringByReplacingPercentEscapesUsingEncoding:NSUTF8StringEncoding];
 
-    NSMutableDictionary<NSString *, id> *event = [self baseEvent];
-    [event addEntriesFromDictionary: @{
+    NSMutableDictionary<NSString *, id> *_event = [self baseEvent];
```

Comment:
Because there is an `event` variable declared on line 314 and I didn't want to reuse it.